jd4/api.py

```diff
@@ -7,48 +7,53 @@
 from jd4.config import cookie_jar, save_cookies
 from jd4.log import logger
 
-SERVER_URL = 'https://vijos.org'
 CHUNK_SIZE = 32768
 
-def full_url(*parts):
-    return urljoin(SERVER_URL, path.join(*parts))
-
 class VJ4Error(Exception):
     def __init__(self, name, message, *args):
         super().__init__('{}: {} ({})'.format(name, message, ', '.join(map(str, args))))
         self.name = name
         self.message = message
         self.args = args
 
+    def __str__(self):
```

Comment:
This should be `__repr__`. `__str__` of an exception should return the message (not verified)